oauth2_http/java/com/google/auth/oauth2/OAuth2Credentials.java

```diff
@@ -97,4 +98,15 @@ private final Long getExpiresInMilliseconds() {
       return (expirationTime.getTime() - clock.currentTimeMillis());
     }
   }
+
+  /**
+   * Override the access token. This is intended primarily to assist with
+   * testing, so that an initial token can be injected.
+   */
+  public void overrideAccessToken(AccessToken accessToken) {
```

Comment:
I think we should instead do this by allowing the access token to be passed into a constructor. We should sync on this, as I have an interim change that does this.